src/components/main-page/About/About.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import styles from "./About.module.scss";
+import { Section, Heading, Text, Card } from "components/ui";
+import { useTranslation, Trans } from "next-i18next";
+
+export function About() {
+  const { t } = useTranslation();
+
+  const paragraphs = t("about.paragraphs", { returnObjects: true }) as string[];
+
+  console.log(paragraphs);
+
+  return (
+    <Section variant="white" id="about">
+      <Heading withAccent>{t("about.title")}</Heading>
+      <div className={styles.textContainer}>
+        {paragraphs?.length > 0 &&
+          paragraphs.map((p: string, index: number) => (
```

Comment:
We could use could use optional chaining here if we're afraid if that's not an array, right?
```suggestion
        {paragraphs?.map((paragraph: string, index: number) => (
```

Reply:
👍